Casks/bose-soundtouch.rb

```diff
@@ -1,22 +1,17 @@
 cask "bose-soundtouch" do
-  version "26.0.0-3251-ff6a93d,st3_2020_a3b99494"
-  sha256 "be74aeaa1e73bd6b07d92ceb5fcff5b58c4023eb7ccdb034fea842080554aa4e"
+  version "27.0.0-3377-1037583,mr4_c4ab85bd"
+  sha256 "b84ee3bbe3350b14873daa003703b745ab24fc0af97615c74a20c6b3e5d87a57"
 
   url "https://downloads.bose.com/ced/soundtouch/#{version.after_comma}/SoundTouch-app-installer-#{version.before_comma}.dmg"
   name "Bose Soundtouch Controller App"
   desc "Control Bose SoundTouch systems from your computer"
   homepage "https://downloads.bose.com/ced/soundtouch/soundtouch_controller_app/index.html"
 
-  # The first-party download page uses JavaScript to render the content. The
-  # URL below redirects to the XML file containing release information that's
-  # used as the data source for the download page.
   livecheck do
-    url "https://worldwide.bose.com/updates/soundtouch"
-    regex(%r{
-      <RELEASE [^>]*?URLPATH=["']/ced/soundtouch/([^/]+)/?["'][^>]*?>\s*
-      <IMAGE [^>]*?FILENAME=["']SoundTouch-app-installer-(\d+(?:\.\d+)+(?:-[a-z\d]+)+)\.dmg["']
-    }ix)
+    url :homepage
+    regex(%r{href=.*/ced/soundtouch/([^/]+)/SoundTouch-app-installer-(\d+(?:\.\d+)+(?:-[a-z\d]+)+)\.dmg})
```

Comment:
```suggestion
    regex(%r{href=.*?/ced/soundtouch/([^/]+)/SoundTouch-app-installer[._-](\d+(?:\.\d+)+(?:-[a-z\d]+)+)\.dmg}i)
```

I went with the same approach as @bevanjkay but the regex needs a few tweaks.